openedx-scalable-ubuntu/azuredeploy.json

```diff
@@ -458,64 +377,47 @@
-      "name": "[concat(variables('lbName'), '/SSH-VM', copyIndex())]",
-      "location": "[resourceGroup().location]",
-      "dependsOn": [
-          "[concat('Microsoft.Network/loadBalancers/',variables('lbName'))]"
-      ],
-      "properties": {
-          "frontendIPConfiguration": {
-              "id": "[variables('frontEndIPConfigID')]"
-          },
-          "protocol": "tcp",
-          "frontendPort": "[copyIndex(2220)]",
-          "backendPort": 22,
-          "enableFloatingIP": false
-      },
-      "copy": {
-          "name": "inboundNatRulesCopy",
-          "count": "[parameters('appVmCount')]"
-      }
-    },
-    {
-      "apiVersion": "[variables('apiVersion')]",
       "type": "Microsoft.Compute/virtualMachines",
-      "name": "[concat(variables('appVmName'),copyIndex())]",
+      "name": "[variables('mySqlVmName')]",
+      "apiVersion": "[variables('computeApiVersion')]",
       "location": "[resourceGroup().location]",
       "dependsOn": [
-        "[concat('Microsoft.Storage/storageAccounts/', variables('storageAccountName'))]",
-        "[concat('Microsoft.Network/networkInterfaces/', variables('appVmName'), copyIndex(), '-nic')]",
-        "[concat('Microsoft.Compute/availabilitySets/', variables('availabilitySetName'))]"
+        "[concat('Microsoft.Storage/storageAccounts/', variables('dbStorageAccountName'))]",
+        "[concat('Microsoft.Network/networkInterfaces/', variables('mySqlVmName'), '-nic')]"
       ],
       "properties": {
-        "availabilitySet": {
-          "id": "[resourceId('Microsoft.Compute/availabilitySets',variables('availabilitySetName'))]"
-        },
         "hardwareProfile": {
-          "vmSize": "[parameters('appVmSize')]"
+          "vmSize": "[parameters('mySqlVmSize')]"
         },
         "osProfile": {
-          "computername": "[concat(variables('appVmName'),copyIndex())]",
+          "computername": "[variables('mySqlVmName')]",
           "adminUsername": "[parameters('adminUsername')]",
           "adminPassword": "[parameters('adminPassword')]"
         },
         "storageProfile": {
-          "imageReference": {
-            "publisher": "[variables('osImagePublisher')]",
-            "offer": "[variables('osImageOffer')]",
-            "sku": "[variables('osImageSKU')]",
-            "version": "latest"
-          },
+          "imageReference": "[variables('imageReference')]",
           "osDisk": {
-            "name": "[concat(variables('appVmName'),copyIndex(),'-osdisk')]",
+            "name": "[variables('mySqlOSDiskName')]",
             "vhd": {
-              "uri": "[concat('http://', variables('storageAccountName'), '.blob.core.windows.net/', variables('vhdBlobContainer'), '/', variables('appVmName'), copyIndex(), '-disk.vhd')]"
+              "uri": "[concat('https://', variables('dbStorageAccountName'), '.blob.core.windows.net/', variables('mySqlVhdContainerName'), '/', 'disk.vhd')]"
```

Comment:
Please see the "all clouds" uri best practice: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md